### PR TITLE
Adding a translation-cache test

### PR DESCRIPTION
**Pull request by jerinphilip**

### CODE REVIEW

File: speed-tests/test_cache_overhead.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  echo "Skipping, hardware doesn't have enough CPUs to run ${THREADS} threads."
+  exit 100
+fi;
+
+
+INPUT_FILE="$BRT_DATA/wngt20/sources.shuf"
+
+
+function benchmark-parameterized-by-cache {
+    CACHE_ARG="$1"
```
Comment (XapaJIaMnu) [inline speed-tests/test_cache_overhead.sh:50]: I see references to CACHE_ARG, but I don't see an example string of what those are? Did I just miss them?

**Review by XapaJIaMnu — DISMISSED** [review]
What are CACHE_ARG?

File: speed-tests/test_cache_overhead.sh (hunk 1 above)
Comment (jerinphilip) [reply speed-tests/test_cache_overhead.sh:50]: true/false. Whether cache is active or not.

Comment (XapaJIaMnu) [reply speed-tests/test_cache_overhead.sh:50]: You're not exposing any of the l4 hyperparameters?

Comment (jerinphilip) [reply speed-tests/test_cache_overhead.sh:50]: It's exposed, I'm not (currently) doing sweep on those. You seem to have recommended one, need we try more?

Comment (XapaJIaMnu) [reply speed-tests/test_cache_overhead.sh:50]: I just want to see the buckets swept a bit, everything else is not essential. I think

**Review by XapaJIaMnu — APPROVED** [review]

**Pull request closed without merging**